doc/phpactor.txt

```diff
@@ -10,6 +10,7 @@
 5. Configuration                |phpactor-configuration|
 6. Requirements                 |phpactor-requirements|
 7. Completion                   |phpactor-completion|
+8. Completion plugins           |phpactor-completion-plugins|
 8. Context Menu                 |phpactor-context-menu|
```

Comment:
`Context Menu` now needs to be `9` :see_no_evil:

Reply:
Damn :) Fixed.